x-pack/platform/plugins/shared/security/server/authentication/authentication_service.ts

```diff
@@ -220,19 +221,26 @@ export class AuthenticationService {
       // Additionally, if logout fails for any reason, we also want to show an error page.
       // At this point we redirect users to the login page if it's available, or render a dedicated unauthenticated error page.
       if (!isLoginPageAvailable || isLogoutRoute) {
-        const customBrandingValue = await customBranding.getBrandingFor(request, {
-          unauthenticated: true,
-        });
+        const location = http.basePath.prepend(
+          `/security/unauthenticated?next=${encodeURIComponent(originalURL)}`
+        );
+        const body = dedent`
+          <html>
```

Comment:
__nit:__ My HTML linter says we need `lang="en"` on `html` element 🙂 That reminds me that we'd need to localize that message too (not that it's super important in this specific case, but still).